flanker/mime/message/part.py

```diff
@@ -684,7 +681,7 @@ def _base64_decode(s):
         return base64.b64decode(s)
 
     except TypeError:
-        s = s.replace("\n", "").replace("\r", "")
+        s = s.translate(_b64_translation_table).replace(LF, "")
```

Comment:
Rather then turning invalid base64 characters to LF and then replacing them with `""` you can just put then in `deletechars` parameter of `translate`:
```python
s = s.translate(_b64_translation_table, _b64_invalid_chars)
```